### PR TITLE
Update phan/phan and drenso/phan-extensions to latest

### PR DESCRIPTION
**Pull request by dairbuirabass**
## Description
Update phan/phan to latest (5.3.0) and drenso/phan-extensions to latest (3.5.1)

## Related Issue
-

## Motivation and Context
Keeping dev packages up to date

## How Has This Been Tested?
Locally

## Checklist:
- [x] My code follows the code style of this project.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] I have assigned the correct milestone or created one if non-existent.
- [x] I have correctly labeled this pull request.
- [ ] I have linked the corresponding issue in this description.
- [x] I have updated the corresponding Jira ticket.
- [x] I have requested a review from at least 2 reviewers
- [x] I have checked the base branch of this pull request
- [x] I have checked my code for any possible security vulnerabilities

### CODE REVIEW

File: phan.php [hunk 1]
```diff
@@ -82,6 +82,10 @@
     ],
     'suppress_issue_types' => [
         'PhanParamSignatureMismatch',
+        'PhanTypeMismatchReturn',
```
Comment (mridang) [inline phan.php:85]: Let's not add more suppressions. I would rather make the fixes. If you are unsure on how to create stubs - let me know.

**Review by mridang — CHANGES_REQUESTED** [review]
See comments.

File: phan.php (hunk 1 above)
Comment (dairbuirabass) [reply phan.php]: Removed suppressions

**Review by mridang — APPROVED** [review]

File: Controller/Adminhtml/Account/Create.php [hunk 2]
```diff
@@ -144,8 +144,8 @@ public function execute()
                 $emailAddress = $this->_request->getParam('email');
                 $accountOwner = $this->nostoOwnerBuilder->build();
                 if (Zend_Validate::is($emailAddress, 'EmailAddress')) {
-                    $accountOwner->setFirstName(null);
-                    $accountOwner->setLastName(null);
+                    $accountOwner->setFirstName('');
+                    $accountOwner->setLastName('');
```
Comment (dairbuirabass) [inline Controller/Adminhtml/Account/Create.php:148]: @olsi-qose is it ok to change from null to empty string here?

**Pull request merged**